app/main.py

```diff
@@ -1,3 +1,13 @@
 def cache(func):
-    # Write your code here
-    pass
+
+    long_time_list = {}
+
+    def inner(*args, **kwargs):
+
+        if tuple(args) in long_time_list:
```

Comment:
there is no need to convert `args` into `tuple` because `args` is already a `tuple`